<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectWidget</class>
 <widget class="QWidget" name="ConnectWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>300</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 14px;/*字体*/
color:rgba(0, 0, 0, 190);/*字体颜色*/</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>服务器ip地址：</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_conn_btn">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>610</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*background-color: rgba(100, 225, 100, 190);/*背景色*/
background-color: rgb(204, 136, 101);
border-style: outset; /*边框样式*/
border-width: 4px;  /*边框宽度为4像素*/
border-radius: 10px; /*边框圆角半径*/
border-color: rgba(255, 225, 255, 30);/*边框颜色*/
font: bold 14px;/*字体*/
color:rgba(0, 0, 0, 100);/*字体颜色*/
padding: 6px; /*填衬*/</string>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>380</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 14px;/*字体*/
color:rgba(0, 0, 0, 190);/*字体颜色*/</string>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>900</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="img.qrc">:/background/resoure/img/connect_bg.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="_ip_input">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>300</y>
     <width>290</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="_username_input">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>380</y>
     <width>290</width>
     <height>40</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>_bg</zorder>
  <zorder>label</zorder>
  <zorder>_conn_btn</zorder>
  <zorder>label_2</zorder>
  <zorder>_ip_input</zorder>
  <zorder>_username_input</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>_ip_input</tabstop>
  <tabstop>_username_input</tabstop>
  <tabstop>_conn_btn</tabstop>
 </tabstops>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
